packages/vue/package.json

```diff
@@ -62,7 +62,7 @@
     "@vueuse/core": "^8.4.2",
     "@vueuse/rxjs": "^8.2.0",
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.3-alpha.1",
+    "@web3-onboard/core": "^2.21.3-alpha.2",
```

Comment:
This version change isn't necessary - please revert

Reply:
fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx